### PR TITLE
Fix wording of spinner readability rules in osu!catch RC

### PR DESCRIPTION
**Pull request by cl8n**
https://osu.ppy.sh/community/forums/topics/1144293

### CODE REVIEW

**Comment by peppy** [comment]
This is no clearer to me. Where is the definition of "active object"? I read the thread but still don't really understand..

**Comment by dorothy3242** [comment]
Active object is already in the catch criteria's glossary, so it was not changed in the pull request.

**Comment by peppy** [comment]
That's *such* a shit term. Ask 10 people what it means and 0 will know.

**Comment by peppy** [comment]
This is not getting merged, by the way. The term needs to change right here and now.

**Review by peppy — CHANGES_REQUESTED** [review]
decide on better term

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ Difficulty-specific rules and guidelines do only apply to the difficulty level t
 #### Rules
 
 - **Dashes and hyperdashes of any kind are disallowed.** This is to ensure an easy starting experience to beginner players. In order to test that out, it must be possible to achieve an SS rank on the difficulty without making use of the dash key.
-- **At least 250 ms must be left between objects and the start and end of spinners to ensure readability.** As an example, a 1/3 gap would be required at 80 BPM, whereas a 1/2 gap would be required at 120 BPM.
+- **At least 250 ms must be left between active objects and the start and end of spinners to ensure readability.** As an example, a 1/3 gap would be required at 80 BPM, whereas a 1/2 gap would be required at 120 BPM.
```
Comment (O1CDeif) [inline wiki/Ranking_Criteria/osu!catch/en.md:81]:
```suggestion
- **At least 250 ms must be left between circles/sliders and the start and end of spinners to ensure readability.** As an example, a 1/3 gap would be required at 80 BPM, whereas a 1/2 gap would be required at 120 BPM.
```

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ Difficulty-specific rules and guidelines do only apply to the difficulty level t
 - **Dashes may be used if the time between the ticks of the desired snapping is 125 ms or higher.** As an example, 1/3 dashes would be allowed at 160 BPM and below, whereas 1/4 dashes would be allowed at 120 BPM and below.
 - **Edge dashes must not be used** as they require extremely precise timing which cannot be expected of less-experienced players.
 - **Basic dashes must not be used more than two times between consecutive fruits. If higher-snapped dashes are used, they must be used singularly (not in conjunction with other dashes).**
-- **At least 250 ms must be left between objects and the start and end of spinners to ensure readability.** As an example, a 1/3 gap would be required at 80 BPM, whereas a 1/2 gap would be required at 120 BPM.
+- **At least 250 ms must be left between active objects and the start and end of spinners to ensure readability.** As an example, a 1/3 gap would be required at 80 BPM, whereas a 1/2 gap would be required at 120 BPM.
```
Comment (O1CDeif) [inline wiki/Ranking_Criteria/osu!catch/en.md:102]:
```suggestion
- **At least 250 ms must be left between circles/sliders and the start and end of spinners to ensure readability.** As an example, a 1/3 gap would be required at 80 BPM, whereas a 1/2 gap would be required at 120 BPM.
```

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 3]
```diff
@@ -126,8 +126,8 @@ Difficulty-specific rules and guidelines do only apply to the difficulty level t
 - **Dashes may be used if the time between the ticks of the desired snapping is 62 ms or higher.** As an example, 1/6 dashes would be allowed at 160 BPM and below, whereas 1/8 dashes would be allowed at 120 BPM and below.
 - **Basic dashes must not be used more than four times between consecutive fruits. Higher-snapped dashes can be used up to two times between consecutive fruits, provided there isn't a direction change between them.**
 - **Edge dashes must not be used** as they require extremely precise timing which cannot be expected of less-experienced players.
-- **At least 125 ms must be left between objects and the start of spinners to ensure readability.** As an example, a 1/4 gap would be required at 120 BPM, whereas a 1/3 gap would be required at 160 BPM.
-- **At least 250 ms must be left between objects and the end of spinners to ensure readability.** As an example, a 1/3 gap would be required at 80 BPM, whereas a 1/2 gap would be required at 120 BPM.
+- **At least 125 ms must be left between active objects and the start of spinners to ensure readability.** As an example, a 1/4 gap would be required at 120 BPM, whereas a 1/3 gap would be required at 160 BPM.
```
Comment (O1CDeif) [inline wiki/Ranking_Criteria/osu!catch/en.md:129]:
```suggestion
- **At least 125 ms must be left between circles/sliders and the start of spinners to ensure readability.** As an example, a 1/4 gap would be required at 120 BPM, whereas a 1/3 gap would be required at 160 BPM.
```

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 4]
```diff
@@ -126,8 +126,8 @@ Difficulty-specific rules and guidelines do only apply to the difficulty level t
...
+- **At least 250 ms must be left between active objects and the end of spinners to ensure readability.** As an example, a 1/3 gap would be required at 80 BPM, whereas a 1/2 gap would be required at 120 BPM.
```
Comment (O1CDeif) [inline wiki/Ranking_Criteria/osu!catch/en.md:130]:
```suggestion
- **At least 250 ms must be left between circles/sliders and the end of spinners to ensure readability.** As an example, a 1/3 gap would be required at 80 BPM, whereas a 1/2 gap would be required at 120 BPM.
```

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 5]
```diff
@@ -149,7 +149,7 @@ Difficulty-specific rules and guidelines do only apply to the difficulty level t
 
 - **Hyperdashes and dashes may be used if the time between the ticks of the desired snapping is 62 ms or higher.** As an example, 1/6 hyperdashes and dashes would be allowed at 160 BPM and below, whereas 1/8 hyperdashes and dashes would be allowed at 120 BPM and below.
 - **Basic hyperdashes must not be used more than four times between consecutive fruits.** If higher-snapped hyperdashes are used, they must not be used in conjunction with other hyperdashes or higher-snapped dashes.
-- **At least 125 ms must be left between objects and the start and end of spinners to ensure readability.** As an example, a 1/4 gap would be required at 120 BPM, whereas a 1/3 gap would be required at 160 BPM.
+- **At least 125 ms must be left between active objects and the start and end of spinners to ensure readability.** As an example, a 1/4 gap would be required at 120 BPM, whereas a 1/3 gap would be required at 160 BPM.
```
Comment (O1CDeif) [inline wiki/Ranking_Criteria/osu!catch/en.md:152]:
```suggestion
- **At least 125 ms must be left between circles/sliders and the start and end of spinners to ensure readability.** As an example, a 1/4 gap would be required at 120 BPM, whereas a 1/3 gap would be required at 160 BPM.
```

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 6]
```diff
@@ -169,8 +169,8 @@ Difficulty-specific rules and guidelines do only apply to the difficulty level t
 
 #### Rules
 
-- **At least 62 ms must be left between objects and the start of spinners to ensure readability.** As an example, a 1/6 gap would be required at 160 BPM, whereas a 1/4 gap would be required at 240 BPM.
-- **At least 125 ms must be left between objects and the end of spinners to ensure readability.** As an example, a 1/4 gap would be required at 120 BPM, whereas a 1/3 gap would be required at 160 BPM.
+- **At least 62 ms must be left between active objects and the start of spinners to ensure readability.** As an example, a 1/6 gap would be required at 160 BPM, whereas a 1/4 gap would be required at 240 BPM.
```
Comment (O1CDeif) [inline wiki/Ranking_Criteria/osu!catch/en.md:172]:
```suggestion
- **At least 62 ms must be left between circles/sliders and the start of spinners to ensure readability.** As an example, a 1/6 gap would be required at 160 BPM, whereas a 1/4 gap would be required at 240 BPM.
```

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 7]
```diff
@@ -169,8 +169,8 @@ Difficulty-specific rules and guidelines do only apply to the difficulty level t
...
+- **At least 125 ms must be left between active objects and the end of spinners to ensure readability.** As an example, a 1/4 gap would be required at 120 BPM, whereas a 1/3 gap would be required at 160 BPM.
```
Comment (O1CDeif) [inline wiki/Ranking_Criteria/osu!catch/en.md:173]:
```suggestion
- **At least 125 ms must be left between circles/sliders and the end of spinners to ensure readability.** As an example, a 1/4 gap would be required at 120 BPM, whereas a 1/3 gap would be required at 160 BPM.
```

**Review by O1CDeif — CHANGES_REQUESTED** [review]
Using just "circles/sliders" to specify those cases would be more clear for the casual reader, as they are much more common terms than "active objects". That'd also formally allow the spinner-spinner situation, which doesn't have an impact in gameplay or combo, but only alter the banana density under certain conditions. Discussion on using "circles/sliders" was taken within the NAT and the change was accepted by the catch BNs after a highlight.

**Comment by peppy** [comment]
in lazer we call them "fruit" and "juice streams", for reference

**Comment by O1CDeif** [comment]
Until lazer is stable enough to be used by mappers/modders (as in, having a functional editor) I'd rather stick to the terms I proposed. When the moment comes, we can adjust the Ranking Criteria and revamp it with the new terminology.

**Comment by peppy** [comment]
circles and sliders works fine for now, sure.

**Comment by cl8n** [comment]
"active object" can't be replaced by "circles/sliders" while keeping same meaning for some other places in RC, I'm not gonna attempt to fix that in this PR and I don't even play catch o_ o

**Review by peppy — APPROVED** [review]

**Pull request merged**